src/BloomExe/ImageProcessing/RuntimeImageProcessor.cs

```diff
@@ -96,10 +96,6 @@ public string GetPathToResizedImage(string originalPath, bool getThumbnail = fal
 			if (new[] {"/img/", "placeHolder", "Button"}.Any(s => originalPath.Contains(s)))
 				return originalPath;
 
-			// don't try to resize .svg files
-			if (originalPath.EndsWith(".svg", StringComparison.OrdinalIgnoreCase))
-				return originalPath;
-
```

Comment:
I assume we still don't want to downsize SVGs...otherwise they would be added to the list of types that "canBeDowngraded"...so why is this check now obsolete?